usr/src/man/man1m/zpool.1m

```diff
@@ -595,6 +599,42 @@ blocked.
 .It Sy panic
 Prints out a message to the console and generates a system crash dump.
 .El
+.It Sy autotrim Ns = Ns Sy on Ns | Ns Sy off
+When set to
+.Sy on Ns , while deleting data, ZFS will inform the underlying vdevs of any
+blocks that have been marked as freed. This allows thinly provisioned vdevs to
+reclaim unused blocks. Currently, this feature supports sending SCSI UNMAP
+commands to SCSI and SAS disk vdevs, SATA TRIM commands to SATA disk vdevs, and
+using file hole punching on file-backed vdevs. The default setting for this
+property is
+.Sy off .
+.Pp
+Please note that automatic trimming of data blocks can put significant stress
+on the underlying storage devices if they do not handle these commands in a
+background, low-priority manner. In that case, it may be possible to achieve
+most of the benefits of trimming free space on the pool by running a manual
+trim every once in a while during a maintenance window using the
+.Nm zpool Cm trim
+command.
+.Pp
+Automatic trim does not reclaim blocks after a delete immediately. Instead,
+it waits approximately 32-64 TXGs (or as defined by the
```

Comment:
Here's the working I put here to address @ahrens comment that this would be the only place where txgs are mentioned in the user documentation:
```Automatic trim does not reclaim blocks after a delete immediately.
Instead, it waits approximately 2-4 minutes to allow for more efficient
aggregation of smaller portions of free space into fewer larger regions,
as well as to allow for longer pool corruption recovery via \fBzpool
import -F\fR.
```
In ZoL we have a separate man page for module parameters into which I dropped the following:
```
\fBzfs_txgs_per_trim\fR (int)
.ad
.RS 12n
Number of transaction groups over which device-specific TRIM commands
are batched when \fBzfs_trim\fR is set.
.sp
Default value: \fB32\fR.
```